contracts/toCKB-typescript/src/utils/types/error.rs

```diff
@@ -12,6 +12,13 @@ pub enum Error {
     TxInvalid,
     LotSizeInvalid,
     PledgeInvalid,
+    KindMismatch = 100,
+    BTCLotSizeMismatch,
+    ETHLotSizeMismatch,
+    UserLockScriptHashMismatch,
+    StatusInvalid,
+    CollateralMismatch,
+    InvalidAddress,
```

Comment:
how about only use  three errors: `CollateralInvalid`, `InvariantDataMutated`, `XChainAddressInvalid` ?

Reply:
I thought more error definition can make it easier to locate errors.